pom.xml

```diff
@@ -2,50 +2,47 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.rdfhdt</groupId>
   <artifactId>hdt-mr</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>2.0</version>

```

Comment:
OK. We'll probably want to reconsider the 2.0 versioning in case we were to publish the project on Maven Central, but I'll go ahead and merge for now.